source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -53,6 +53,8 @@ struct ExtProcFilterStats {
   ALL_EXT_PROC_FILTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+inline constexpr absl::string_view ImmediateResponsePrefix = "direct_response_from_ext_proc[{}]";
```

Comment:
This need to be in the xDS config, it's highly likely folks will want some control over this.

Reply:
Also, it's not a prefix, it's basically a local response header template. Shouldn't we use CER for this or something similar? At least similar format for standard response header templating.

Reply:
Its not sending error to user. This is in the access log (%response_details%). Essentially, all this is to make this available under a well defined key in the access log filter.

Reply:
This is simply adding a structure to the access log response details so that access log filters can differentiate the ext_proc response from the rest. As the proto says

```
// A string detailing why this local reply was sent, which may be included
  // in log and debug output (e.g. this populates the %RESPONSE_CODE_DETAILS%
  // command operator field for use in access logging).
  string details
```